packages/fiori/src/IllustratedMessage.js

```diff
@@ -222,6 +222,19 @@ const metadata = {
 		subtitle: {
 			type: HTMLElement,
 		},
+		/**
+		 * Defines the title of the component.
+		 * <br />
```

Comment:
Minor, but let's leave these on a single line and without slashes, just to be consistent - `<br><br>`.